impeller/typographer/backends/stb/typeface_stb.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include "flutter/fml/macros.h"
+#include "flutter/fml/mapping.h"
+#include "impeller/base/backend_cast.h"
+#include "impeller/typographer/typeface.h"
+#include "third_party/stb/stb_truetype.h"
+
+namespace impeller {
+
+class TypefaceSTB final : public Typeface,
+                          public BackendCast<TypefaceSTB, Typeface> {
+ public:
+  // "Typical" conversion from font Points to Pixels.
+  // This assumes a constant pixels per em.
+  static constexpr float kPointsToPixels = 96.0 / 72.0;
+
+  TypefaceSTB() = delete;
```

Comment:
ool this dtor.

Reply:
Removed this because it's already implicitly deleted.